packages/font-support-info/src/index.ts

```diff
@@ -0,0 +1,1090 @@
+      'que',
+      'roh',
+      'run',
+      'sag',
+      'gla',
+      'sna',
+      'som',
+      'spa',
+      'swh',
+      'swe',
+      'zul',
+    ],
+  },
+  noto_regular: {
+    name: 'Noto Regular',
+    support_languages: [
+      'azj',
+      'bel',
+      'bos',
+      'bul',
+      'mkd',
+      'rus',
+      'srp',
+      'ukr',
+      'hin',
+      'mar',
+      'nep',
+      'ell',
+      'afr',
+      'aka',
+      'ast',
+      'eus',
+      'bem',
+      'bre',
+      'cat',
+      'hrv',
+      'ces',
+      'dan',
+      'nld',
+      'eng',
+      'pcm',
+      'epo',
+      'est',
+      'ewe',
+      'fao',
+      'fin',
+      'fra',
+      'fur',
+      'glg',
+      'lug',
+      'deu',
+      'haw',
+      'hun',
+      'ibo',
+      'ind',
+      'gle',
+      'ita',
+      'dyo',
+      'kea',
+      'kal',
+      'kin',
+      'lav',
+      'lin',
+      'lit',
+      'ltz',
+      'kde',
+      'mlt',
+      'glv',
+      'sme',
+      'nno',
+      'nyn',
+      'pol',
+      'por',
+      'que',
+      'ron',
+      'roh',
+      'run',
+      'sag',
+      'gla',
+      'srp',
+      'sna',
+      'slk',
+      'som',
+      'spa',
+      'swh',
+      'swe',
+      'tur',
+      'hsb',
+      'vai',
+      'vie',
+      'cym',
+      'fry',
+      'yor',
+      'zul',
+    ],
+  },
+  playfair: {
+    name: 'Playfair',
+    support_languages: [
+      'afr',
+      'eus',
+      'bem',
+      'bre',
+      'cat',
+      'dan',
+      'nld',
+      'eng',
+      'pcm',
+      'fao',
+      'fra',
+      'fur',
+      'glg',
+      'deu',
+      'ind',
+      'gle',
+      'ita',
+      'kea',
+      'kin',
+      'ltz',
+      'kde',
+      'glv',
+      'nno',
+      'nyn',
+      'por',
+      'que',
+      'roh',
+      'run',
+      'sag',
+      'gla',
+      'sna',
+      'som',
+      'spa',
+      'swh',
+      'swe',
+      'zul',
+    ],
+  },
+  raleway: {
+    name: 'Raleway',
+    support_languages: [
+      'afr',
+      'eus',
+      'bem',
+      'bre',
+      'cat',
+      'dan',
+      'nld',
+      'eng',
+      'pcm',
+      'fao',
+      'fra',
+      'fur',
+      'glg',
+      'deu',
+      'ind',
+      'gle',
+      'ita',
+      'kea',
+      'kin',
+      'ltz',
+      'kde',
+      'glv',
+      'nno',
+      'nyn',
+      'por',
+      'que',
+      'roh',
+      'run',
+      'sag',
+      'gla',
+      'sna',
+      'som',
+      'spa',
+      'swh',
+      'swe',
+      'zul',
+    ],
+  },
+};
+
+export class FontLanguageInfo {
+  protected fonts: { [font: string]: IFontInfoData };
+
+  constructor() {
+    this.fonts = FONTS_INFO_LIST;
+  }
+
+  /**
+   * Check provided font family supported with
+   * given language code and if support returns true
+   * else returns false
+   * @param fontFamily font family
+   * @param languageCode language code
+   */
+  public isFontSupport(fontFamily: string, languageCode: string): boolean {
+    if (!this.fonts[fontFamily]) {
```

Comment:
This is case-sensitive right? If the user passes `noto regular`, it will return false right? We better make this non-case-sensitive. And also, in svg data the font comes as `playfair` if I select the italic option also. How do you interpret this and match here?

Reply:
I checked with fonts, language support does not depend on styles ( bold, italic ).